src/main/java/com/epam/ta/reportportal/core/filter/IShareUserFilterHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.ta.reportportal.core.filter;
+
+import com.epam.ta.reportportal.auth.ReportPortalUser;
+import com.epam.ta.reportportal.commons.querygen.Filter;
+import com.epam.ta.reportportal.ws.model.filter.UserFilterResource;
+import org.springframework.data.domain.Pageable;
+
+public interface IShareUserFilterHandler {
```

Comment:
let's get rid of "I" prefix

Reply:
It will be inconsistent with IGetUserFilterHandler, IUpdateUserFilterHandler, IDeleteUserFilterHandler, ICreateUserFilterHandler and others